build.sbt

```diff
@@ -97,8 +97,10 @@ lazy val storage = project
     libraryDependencies ++= Seq(
       "dev.zio"                %% "zio"                     % "1.0.0-RC18-2",
       "dev.zio"                %% "zio-interop-guava"       % "28.2.0.1",
-      "com.google.cloud"       % "google-cloud-storage"     % "1.108.0",
+      "com.google.cloud"        % "google-cloud-storage"    % "1.108.0",
       "org.scala-lang.modules" %% "scala-collection-compat" % "2.1.6"
     )
   )
   .enablePlugins(BuildInfoPlugin)
+
+resolvers += "google" at "https://maven.google.com/"
```

Comment:
Is this the correct place to put the resolver?

Reply:
You can add them here, or in `BuildHelper`, both should suffice.